aspnetcore/release-notes/aspnetcore-6.0.md

```diff
@@ -0,0 +1,704 @@
+---
+title: What's new in ASP.NET Core 6.0
+author: rick-anderson
+description: Learn about the new features in ASP.NET Core 6.0.
+ms.author: riande
+ms.custom: mvc
+ms.date: 10/29/2021
+no-loc: [Home, Privacy, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR, Kestrel]
+uid: aspnetcore-6.0
+---
+# What's new in ASP.NET Core 6.0
+
+This article highlights the most significant changes in ASP.NET Core 6.0 with links to relevant documentation.
+
+## ASP.NET Core MVC and Razor improvements
+
+## Minimal APIs
+
+Minimal APIs are architected to create HTTP APIs with minimal dependencies. They are ideal for microservices and apps that want to include only the minimum files, features, and dependencies in ASP.NET Core. For more information, see <xref:tutorials/min-web-api>.
+
+## SignalR
+
+### SignalR message size
+
+Size limitations on SignalR message size for Blazor Server apps have been removed.
```

Comment:
Doesn't blazor have its own section that this should be in? Also, I hope there is more than "we removed a limit" because that doesn't sound safe.

Reply:
It can be moved, but there's nothing to cross-link for this in the Blazor node. IIRC, we removed a bit of language and don't (usually) cover migration in an API removal scenario in the reference topics. @TanayParikh can remark on the safety part.

Reply:
What change or documentation is this in reference to? As Brennan mentioned, removing SignalR message size limitations is not something we would be doing due to the security concerns it would bring.

Reply:
https://github.com/dotnet/aspnetcore/issues/30289

Reply:
I commented that out. If the SignalR teams wants to document the message size change, they'll have to add it back it.

Reply:
Ah I see, that issue is in reference to supporting streaming interop in Blazor, thereby allowing for transfer of data larger than the SignalR message size via multiple chunks smaller than the SignalR message size. I've updated that issue's title.

For here, I think we can probably just remove this section.